apps/cowswap-frontend/src/common/pure/ClickableAddress/Content.tsx

```diff
@@ -0,0 +1,71 @@
+import EtherscanImage from '@cowprotocol/assets/cow-swap/etherscan-icon.svg'
+import { UI } from '@cowprotocol/ui'
+
+import SVG from 'react-inlinesvg'
+import styled from 'styled-components/macro'
+
+import CopyHelper from 'legacy/components/Copy'
+
+import { ExtLink } from 'pages/Account/styled'
+
+const Box = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  justify-content: start;
+  gap: 12px;
+  min-width: 124px;
+`
+
+const CopyHelperWrapper = styled(CopyHelper)`
+  opacity: 0.75;
+  transition: opacity var(${UI.ANIMATION_DURATION}) ease-in-out;
+
+  &:hover {
+    opacity: 1;
+  }
+`
+
+const ExtLinkWrapper = styled(ExtLink)`
+  padding-left: 6px;
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  font-size: 0.825rem;
+  opacity: 0.75;
+  transition: opacity var(${UI.ANIMATION_DURATION}) ease-in-out;
+  
+  &:hover {
+    text-decoration: none;
+    opacity: 1;
+  }
+`
+
+const Text = styled.span`
+  margin-left: 8px;
+`
+
+interface ContentProps {
+  address: string
+  target: string
+}
+
+export function Content({ address, target }: ContentProps) {
+  return (
+    <Box>
+      <CopyHelperWrapper toCopy={address}>
+        <Text>Copy address</Text>
+      </CopyHelperWrapper>
+      <ExtLinkWrapper href={target} target="_blank">
+        <SVG
+          src={EtherscanImage}
```

Comment:
Would try to keep our monochrome icons look. Would consider the use of `inline-svg` library we use elsewhere to then set the Etherscan SVG icon color with a styled component.